align_to_reference.py

```diff
@@ -1,57 +0,0 @@
-#!/usr/bin/env python
-
-import subprocess
-import os
-import sys
-
-from config import PICARD_JAR, JAVA
```

Comment:
It's generally advisable to import submodules using the full path: `from mwgs.config import ...` or using relative paths: `from .config import ...`

This way you will avoid potential conflicts that can come up with other packages that might be installed. You will generally need to make your package installable for this to work well.